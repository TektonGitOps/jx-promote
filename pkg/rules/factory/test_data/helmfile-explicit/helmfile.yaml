repositories:
- name: yourorg
  url: https://yourorg.example.com/charts
- name: dev
  url: http://jenkins-x-chartmuseum:8080
releases:
- name: dbmigrator
  labels:
    job: dbmigrator
  chart: ./dbmigrator
