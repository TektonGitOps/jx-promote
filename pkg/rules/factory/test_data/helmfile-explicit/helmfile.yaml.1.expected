helmDefaults:
  atomic: false
  devel: false
  force: false
  recreatePods: false
  tillerless: false
  timeout: 0
  tls: false
  verify: false
  wait: false
releases:
- chart: ./dbmigrator
  labels:
    job: dbmigrator
  name: dbmigrator
- name: myapp
  version: 1.2.3
repositories:
- name: yourorg
  url: https://yourorg.example.com/charts